system/security/securesocket/securesocket.cpp

```diff
@@ -1641,3 +1641,71 @@ SECURESOCKET_API int signCertificate(const char* csr, const char* ca_certificate
 }
 
 }
+
+ISmartSocketException *createSecureSmartSocketException(int errorCode, const char *msg)
+{
+    return new SmartSocketException(errorCode, msg);
+}
```

Comment:
Looks identical to version in jlib? (in jsmartsock)
If so, better to reused that.